cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -72,6 +72,48 @@ message DeleteSurveyResponse {
     bool success = 1;
 }
 
+/**
+    Source Files
+**/
+
+message RegisterSourceSegyFileRequest {
+    Identifier survey = 1;                              // [required] The survey this source file belongs to
+    string path = 2;                                    // [required] Path including protocol, bucket and directory structure. Example: "gs://cognite-seismic-eu/samples/"
+    string name = 3;                                    // [required] Unique filename including extension. Example: "DN1302M03R16_MERGED_KPSDM_00-32_DEG_T.sgy". The name must be unique across buckets and can be used to identify this file in query requests
```

Comment:
I wonder if it's time to relax this name uniqueness requirement and replace path/name with a single cloud storage URI field? Might be painful in the transition from v0 to v1, although we could maintain legacy path and name fields from the URI by just splitting the URI. Or am I being difficult here?

Reply:
good idea =)